toto/runlib.py

```diff
@@ -100,6 +100,11 @@ def record_artifacts_as_dict(artifacts):
     return artifacts_dict
 
   for artifact in artifacts:
+
+    if not os.path.exists(artifact):
+      log.warning("path: {} does not exist, skipping..".format(artifact))
+      continue 
+
```

Comment:
This is a good idea. Shouldn't it also work to have this as an `else` clause [below](https://github.com/in-toto/in-toto/pull/33/files#diff-c8942631fdeb6716abe5f5b4611c90cfR115)? Like so:
```
if is file:
  record
elif is directory:
 record using walk (flattens the tree)
else:
  warn
```